code/modules/reagents/reagent_containers/food.dm

```diff
@@ -120,3 +120,38 @@
 /obj/item/weapon/reagent_containers/food/snacks/tortillachip/New()
 	..()
 	reagents.add_reagent(NUTRIMENT,1)
+
+/obj/item/weapon/poutineocean
+	name = "poutine ocean"
+	desc = "This enormous, horrific cheese wheel is filled to the brim with poutine. You'll probably never reach the bottom."
+	icon = 'icons/obj/food_huge.dmi'
+	icon_state = "poutineocean"
+	pixel_x = -16
+	pixel_y = -8
+	var/type_to_dispense = /obj/item/weapon/reagent_containers/food/snacks/poutine
+	w_class = W_CLASS_GIANT
+
+/obj/item/weapon/poutineocean/attack_hand(mob/user)
```

Comment:
As opposed to an `/obj`.
Doesn't really need to be an item, nor a weapon, if it can't be picked up at all ever.